### ISSUE TITLE
Multiple classes on the same html element are parsed as a single class

### ISSUE THREAD

**Issue by lasse16** [lomirus/html_editor#4]
An html snippet like 
```html
<div class="a b">
</div>
```
 gets parsed into an `Element` like 
```rust
Element { name: "div", attrs: [("class", "a b")], children: [] }
```

There is only one classs attribute which contains both classes seperated by whitespace. 
This leads to queries for a only one of the two classes to fail.

This test fails.
```rust
    #[test]
    fn html_editor_multiple_class_parsing() {
        let test_snippet = r#"<div class="a b"></div>"#;
        let result = parser::parse(test_snippet).unwrap();
        print!("{:?}", result);      
        // This selector fails
        let selector = Selector::from(".a");
       // This selector works
       // let selector = Selector::from(".a b");

        result.query(&selector).unwrap();
        assert!(true);
    }
```

My expected behaviour would be a seperate class attribute for each class inside the html.
In this case:
```rust
Element { name: "div", attrs: [("class", "a"),("class", "b")], children: [] }
```

**Comment by lomirus** [comment]
Hello, Lasse Haffke!

Thank you for reporting, this bug has just been fixed in v0.5.1.

BTW, `Selector::from(".a b")` to be parsed as a class selector for `a b` is weird. This problem will also be fixed soon, replaced with the descendant selector support.

**Comment by lasse16** [comment]
Nice ! 
Thanks for the quick work :)
